doc/source/acb.rst

```diff
@@ -1204,7 +1246,11 @@ Vector functions
 
 .. function:: void _acb_vec_scalar_div_fmpz(acb_ptr res, acb_srcptr vec, slong len, const fmpz_t c, slong prec)
 
-   Performs the respective scalar operation elementwise.
+    Performs the respective scalar operation elementwise.
```

Comment:
extra white space?

Reply:
Yes, the indent was 3 instead of 4 like everywhere else and I thought I'd just fix that.

Reply:
indeed